pkg/users/account-create-dialog.js

```diff
@@ -118,6 +130,23 @@
         return _("The full name must not contain colons.");
 }
 
+function validate_uid(uid, accounts, change) {
+    if (!uid)
+        return _("No UID specified");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18353-20230217-135348-b75b6476-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)